packages/cli/src/transform.js

```diff
@@ -94,16 +100,56 @@ export async function compileFile(
 ): Promise<?string> {
   const inputFilePath = path.join(config.input, filePath);
   const outputFilePath = path.join(config.output, filePath);
+  const cachePath = getDefaultCachePath();
+
+  const inputHash = computeHash(inputFilePath);
+  let oldOutputHash = null;
+  if (fs.existsSync(outputFilePath)) {
+    try {
+      oldOutputHash = computeHash(outputFilePath);
+    } catch (err) {
+      console.error(
```

Comment:
Why do we need the `try-catch` here but not when the hash is being calculated for the input file?

Reply:
whoops leftover from debugging, will rm